SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -60,6 +60,9 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	 * Otherwise return the single worker settings class.  */
 	TSubclassOf<USpatialMultiWorkerSettings> GetMultiWorkerSettingsClass(bool bForceNonEditorSettings = false);
 
+	// This function should only be used to set the multi-worker setting class when the world isn't open in the editor
```

Comment:
Can't we verify that with a `#WITH_EDITOR` check?

Reply:
Hm, like "not-with-editor"? Problem is that the commandlet is using it when compiled with the editor.
The comment is meant to discourage its use at runtime and while the map is open for editing in the editor.
I could do #WITH_EDITOR and check if itsn't the current editor map, but truth is - I don't actually *know* what will happen if it's modified at runtime / during editing.

Reply:
Maybe worth expanding on this in the comment?

Reply:
Basically I think that if this should only be used in a specific scenario when the world isn't in the editor. We should make the function definition do nothing and log an error if called when the scenario isn't true.

Reply:
The behaviour of the commandlet wouldn't work then, I think. The automation utils basically make it as if the world was open for editing in the editor window. So we are kind of discouraging the exact use case we use it for.
We could still compile this out if we are running without the editor, I guess.